src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml

```diff
@@ -1,190 +1,181 @@
-   <translation id="tools_lineDiagnostics_detail_lastSync" qtlid="326028">Dernière Synchronisation</translation>
-   <translation id="tools_lineDiagnostics_detail_contactPhone" qtlid="326041">Numéro de contact</translation>
-   <translation id="tools_lineDiagnostics_detail_operator" qtlid="63052">Opérateur</translation>
-   <translation id="tools_lineDiagnostics_detail_NRAName" qtlid="309477">Nom du NRA</translation>
-   <translation id="tools_lineDiagnostics_detail_NRADistance" qtlid="63040">Distance du NRA</translation>
+   <!-- DIAGNOSTIC V2-->
+   <translation id="tools_lineDiagnostics_detection_step">Détection du type d'incident</translation>
+   <translation id="tools_lineDiagnostics_investigation_step">Diagnostic de votre accès internet</translation>
+   <translation id="tools_lineDiagnostics_solution_proposal_step">Proposition de solutions</translation>
+
+   <!-- COMMON -->
    <translation id="tools_lineDiagnostics_detail_yes" qtlid="34776">Oui</translation>
    <translation id="tools_lineDiagnostics_detail_no" qtlid="34788">Non</translation>
+   <translation id="tools_lineDiagnostics_waitingRobot">Test technique en cours.</translation>
+   <translation id="tools_lineDiagnostics_please_wait">Veuillez patienter</translation>
+   <translation id="tools_lineDiagnostics_hours_to">à</translation>
+   <translation id="tools_lineDiagnostics_validate">Valider</translation>
+   <translation id="tools_lineDiagnostics_error_loading_translations">Le chargement des traductions a échoué</translation>
+
+   <!-- INCIDENT DETECTON STEP -->
+   ￼￼<translation id="tools_lineDiagnostics_detection_step_test">Veuillez patienter, nous testons actuellement votre synchronisation au réseau</translation>
+   <translation id="tools_lineDiagnostics_diagnostic_critical_problem">Un incident est survenu, nous vous prions de nous en excuser. Nous mettons tout en œuvre pour le résoudre au plus vite.<br />N'hésitez pas à réitérer cette manipulation ultérieurement</translation>
+
+   <translation id="tools_lineDiagnostics_diagnostic_run_error">Une erreur est survenue lors du diagnostic</translation>
+   <translation id="tools_lineDiagnostics_diagnostic_cancel_error">Une erreur est survenue lors de l'annulation du diagnostic</translation>
+   <translation id="tools_lineDiagnostics_detection_step_modem_question">Votre modem est-il synchronisé ?</translation>
+   <translation id="tools_lineDiagnostics_detection_step_modem_question_help">Un voyant sur le modem (broadband ou adsl) devrait vous indiquer le statut de la synchronisation. Si ce n'est pas le cas, vous devriez retrouver ce statut de synchronisation sur l'interface du modem.</translation>
+   <translation id="tools_lineDiagnostics_detection_step_warning"><strong>Nous avons détecté un incident sur votre ligne.</strong><br/>Nous allons maintenant lancer un diagnostic de votre ligne pour rétablir votre accès internet rapidement.
```

Comment:
Avoid tags in XML

Reply:
I remove some tags, but some other are still in, because some text are very long and action text are made to be generic See https://github.com/ovh-ux/ovh-angular-line-diagnostic/pull/9/files#diff-e2a647e6d85b862c882dab63e33a5d93R27